modin/pandas/test/test_api.py

```diff
@@ -45,6 +45,7 @@ def test_top_level_api_equality():
     ]
 
     ignore_modin = [
+        "indexing",
```

Comment:
`ignore_modin` here lists sub-modules of `modin.pandas` module, which are not presented in `pandas`, `modin.pandas.indexing` is such type of module. 

Previously this module was never used in the flow of importing `modin.pandas` and so wasn't visible in the `modin.pandas` module scope and hence wasn't listed here. Now we import `modin.pandas.indexing` inside `PandasFramePartition` and so it became visible in `modin.pandas`. That's the reason this line is added.

Reply:
we may want to consider splitting `modin.pandas` in two - one that mimics Pandas API and another that is used to implement the first. cc @YarShev

Reply:
@vnlitvinov , I think we can keep as it is for now.